.github/workflows/cd.yml

```
@@ -50,7 +58,12 @@ jobs:
           username: ${{ secrets.SSH_USERNAME }}
           key: ${{ secrets.SSH_PRIVATE_KEY }}
           script: |
-            echo "${{ secrets.ENV_FILE }}" > /home/ubuntu/actions-runner/_work/artium-be/artium-be/.env
+            if [ "${PROFILE}" = "dev" ]; then
+              echo "${{ secrets.ENV_DEV }}" > /home/ubuntu/actions-runner/_work/artium-be/artium-be/.env
+            elif [ "${PROFILE}" = "prod" ]; then
+              echo "${{ secrets.ENV_PROD }}" > /home/ubuntu/actions-runner/_work/artium-be/artium-be/.env
+            fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)